### PR TITLE
slicking scrape info, managed to pull a join :-)

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

File: shoebox/app/com/keepit/model/ScrapeInfo.scala [hunk 1]
```diff
@@ -73,6 +76,48 @@ case class ScrapeInfo(
...
+
+    def uri = foreignKey("SCRAPE_NURI_FK", uriId, inject[NormalizedURIRepo].table)(_.id)
+  }
+
+  def allActive(implicit session: RSession): Seq[ScrapeInfo] = {
+    val q = (for {
+       Join(s, u) <- table innerJoin inject[NormalizedURIRepoImpl].table on (_.uriId is _.id)
+       if u.state is NormalizedURIStates.INDEXED
+     } yield s.*)
+   println("================================")
```
Comment (2is10) [inline shoebox/app/com/keepit/model/ScrapeInfo.scala:113]: println intentional?

File: shoebox/app/com/keepit/model/ScrapeInfo.scala [hunk 2]
```diff
@@ -73,6 +76,48 @@ case class ScrapeInfo(
...
+    def signature =  column[String]("signature", O.NotNull)
+    def destinationUrl = column[String]("destination_url", O.Nullable)
+    def * = id.? ~ uriId ~ lastScrape ~ nextScrape ~ interval ~ failures ~ state ~ signature ~ destinationUrl.? <> (ScrapeInfo, ScrapeInfo.unapply _)
+
+    def uri = foreignKey("SCRAPE_NURI_FK", uriId, inject[NormalizedURIRepo].table)(_.id)
+  }
+
+  def allActive(implicit session: RSession): Seq[ScrapeInfo] = {
+    val q = (for {
+       Join(s, u) <- table innerJoin inject[NormalizedURIRepoImpl].table on (_.uriId is _.id)
```
Comment (2is10) [inline shoebox/app/com/keepit/model/ScrapeInfo.scala:110]: nice!

File: shoebox/app/com/keepit/model/ScrapeInfo.scala (hunk 1 above)
Comment (eishay) [inline shoebox/app/com/keepit/model/ScrapeInfo.scala:113]: no, kicked it out in the next commit

File: shoebox/app/com/keepit/model/ScrapeInfo.scala (hunk 2 above)
Comment (andrewconner) [inline shoebox/app/com/keepit/model/ScrapeInfo.scala:110]: Yeah. I'm a big fan of Slick's syntax. Very similar to Scala collections.

**Pull request merged**